.github/stale.yml

```
@@ -0,0 +1,41 @@
+# Configuration for probot-stale based on: https://github.com/facebook/react-native/blob/master/.github/stale.yml
+
+ # Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+ # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+daysUntilClose: 7
+
+ # Issues or Pull Requests.
+exemptLabels:
+  - pinned
+  - security
+  - discussion
+
+ # Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+ # Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+ # Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+ # Label to use when marking as stale
+staleLabel: stale
+
+ # Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions. You may also mark this issue as a "discussion" and i
```

```suggestion
    for your contributions. You may also mark this issue as a "discussion" and I
```

Minor nitpick ;)

Aside from that this looks great, thanks for creating this PR ❤️

---

Done! =D